librz/core/cmd/cmd_analysis.c

```diff
@@ -8893,9 +8893,15 @@ RZ_IPI RzCmdStatus rz_analyze_bytes_handler(RzCore *core, int argc, const char *
 }
 
 RZ_IPI RzCmdStatus rz_analyze_n_bytes_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
-	int len = core->blocksize;
-	int tbs = len;
-	len = (int)rz_num_get(core->num, argv[1]);
+	ut32 len = core->blocksize;
+	ut32 tbs = len;
+
+	len = rz_num_get(core->num, argv[1]);
+	if ((int)len <= 0) {
+		eprintf("Invalid zero or negative arguments.\n");
```

Comment:
RZ_LOG_ERROR here. Use `eprintf` only to debug your code, when you push it to the PR, there should not be any.
use RZ_LOG_* functions for logging errors, warning, infos and debug messages and use rz_cons_* for anything that needs to be printed like you would do using printf

Reply:
Thanks for the detailed explanation. :-)